inc/template.php

```diff
@@ -1864,6 +1866,25 @@ function tpl_classes() {
         (isset($INFO) && $INFO['exists']) ? '' : 'notFound',
         ($ID == $conf['start']) ? 'home' : '',
     );
+    $id=$ID;
+    if($id[0] == ':')
+        $id = substr($id,1);
+    $id = str_replace('_', '__', $id);
+    $chain = explode(':', $id);
+    $pgname = array_pop($chain);
+
+    $classes[] = 'lv_' . count($chain);
+    $ns = 'ns_';
+    foreach($chain as $comp) {
+        $ns .= "_$comp";
+        $classes[] = $ns;
+    }
+    $parent = array_pop($chain);
+    $classes[] = "ns_{$parent}_";
+    $classes[] = "pg_$pgname";
+    if('' != $parent)
+        $classes[] = 'pg_' . str_replace(':', '_', $id);
```

Comment:
Please wrap the statement of the if with {}

Reply:
it's done in #0a82ce1